pkg/api/configsync/v1alpha1/zz_generated.conversion.go

```diff
@@ -617,21 +617,22 @@ func Convert_v1alpha1_OverrideSpec_To_v1beta1_OverrideSpec(in *OverrideSpec, out
 	return autoConvert_v1alpha1_OverrideSpec_To_v1beta1_OverrideSpec(in, out, s)
 }
 
+// Convert_v1beta1_OverrideSpec_To_v1alpha1_OverrideSpec is written manually because of lossy conversion
+func Convert_v1beta1_OverrideSpec_To_v1alpha1_OverrideSpec(in *v1beta1.OverrideSpec, out *OverrideSpec, s conversion.Scope) error {
```

Comment:
This custom func needs to be in the conversion.go file because it's not generated. This whole file gets replaced when re-generated.

Reply:
moved to conversion.go